Dimensions.py

```diff
@@ -841,14 +848,15 @@ def _new_tube_cb(self, id, initiator, type, service, params, state):
                       'params=%r state=%d', id, initiator, type, service,
                       params, state)
 
-        if (type == telepathy.TUBE_TYPE_DBUS and service == SERVICE):
-            if state == telepathy.TUBE_STATE_LOCAL_PENDING:
+        if (type == TUBE_TYPE_DBUS and service == SERVICE):
+            # TODO: not sure if this is correct constant
```

Comment:
I've checked, it seems correct.  It is an enum evaluating to zero, in both old and new bindings.

Reply:
Yeah, thanks. Fixed in 7dca42e.